src/components/Map/map.js

```diff
@@ -60,10 +60,28 @@ class Map extends Component {
       latitude: coordinates[1]
     };
 
+    let forwardGeocoder = (query) => {
+      let { providers } = this.props;
+      var matchingFeatures = [];
+      const geoJSONProviders = this.geoJSONFeatures(providers.byId);
+      for (var i = 0; i < geoJSONProviders.length; i++) {
+      var feature = geoJSONProviders[i];
```

Comment:
Need to fix the indentation here and below

Reply:
How do people feel about a linter? It could keep at the very least things like this cleaner and/or future people that dabble to adhere to some established rule. Like don't even bother looking at someone's pull request until the linter passes.